presto-main/src/main/resources/webapp/src/utils.js

```diff
@@ -446,6 +446,16 @@ export function parseDataSize(value: string): ?number {
     }
 }
 
+export function parseAndFormatDataSize(value: string): string {
+    const parsed = parseDataSize(value);
+
+    if (parsed == null) {
+        return "";
```

Comment:
Previously we would just call `formatDataSize` which doesn't have an `if` like that. So it looks not needed.

I guess this is defensive approach preventing printing something like "null" or "undefined" in UI (which looks very cheap), so I like that nonetheless. Please add a rationale comment.

Reply:
parseDataSize returns number? so it cannot be passed directly to formatDataSize (it takes number)